src/main/fc/fc_rc.c

```diff
@@ -219,8 +227,10 @@ FAST_CODE uint8_t processRcInterpolation(void)
         if (isRXDataNew && rxRefreshRate > 0) {
             rcInterpolationStepCount = rxRefreshRate / targetPidLooptime;
 
-            for (int channel = ROLL; channel < interpolationChannels; channel++) {
-                rcStepSize[channel] = (rcCommand[channel] - rcCommandInterp[channel]) / (float)rcInterpolationStepCount;
+            for (int channel = ROLL; channel <= THROTTLE; channel++) {
+                if ((1 << channel) & interpolationChannels) {
+                    rcStepSize[channel] = (rcCommand[channel] - rcCommandInterp[channel]) / (float)rcInterpolationStepCount;
```

Comment:
Need to review this one in disasm again, the change could have prevented the reciprocity optimization.